src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { memo } from 'react';
+import cN from 'classnames';
+
+type Props = {
+  amountTodos: number,
+  filterStatus: string,
+  selectFilter: (event: React.MouseEvent) => void,
```

Comment:
Function name `selectFilter` and the fact it gets raw `event` as a parameter are on different abstraction levels